web/src/test/scala/quasar/api/Gen.scala

```diff
@@ -0,0 +1,11 @@
+package quasar.api
+
+import org.scalacheck.Arbitrary
+import quasar.Predef._
+import quasar.fs.{Natural, Positive}
+
+object Gen {
```

Comment:
Can we move these to `core` tests under the `quasar.fs` package where `Natural` and `Positive` are defined?